PowerShell/JumpCloud Commands Gallery/Linux Commands/Linux - Set ReleaseChannel In JumpCloud Password Manager.md

```diff
@@ -0,0 +1,45 @@
+#### Name
+
+Linux - Set JumpCloud Password Manager's Release Channel | v1.0 JCCG
+
+#### commandType
+
+linux
+
+#### Command
+
+```
+#!/bin/bash
+
+# Set $RELEASE_CHANNEL to beta OR dogfood OR public ON LINE 4 depending on your desired release channel
+
+RELEASE_CHANNEL="public"
```

Comment:
We can't expect admins to do the right thing, it would probably be a good idea to add a check to ensure that `$RELEASE_CHANNEL` is one of 'public','dogfood','beta'